apps/web/package.json

```
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "eleventy",
+    "build": "eleventy && rm -rf ../../_site && mv _site ../../",
```

```suggestion
    "build": "eleventy",
```

Since the default is `_site`, I would expect this not to work unless you keep the output directory as the default